versioned/storage/bigtable/src/main/java/org/projectnessie/versioned/storage/bigtable/BigTableConstants.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2023 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.versioned.storage.bigtable;
+
+import com.google.protobuf.ByteString;
+
+final class BigTableConstants {
+
+  static final String TABLE_REFS = "refs";
+  static final String TABLE_OBJS = "objs";
+
+  static final String FAMILY_REFS = "r";
+  static final String FAMILY_OBJS = "o";
+
+  static final ByteString QUALIFIER_OBJS = ByteString.copyFromUtf8("o");
+  static final ByteString QUALIFIER_REFS = ByteString.copyFromUtf8("r");
+
+  // Tue Apr 7 08:14:21 2020 +0200
+  static final long CELL_TIMESTAMP = 1586232861000L;
```

Comment:
Out of curiosity, what is the rationale for using a fixed timestamp for all cells?

Reply:
BIgTable stores the _history_ of all cells with the timestamp. There are two "solutions" for this:
1. set a constant timestamp (so there's always just one cell-version) (as in the previous state of this PR)
2. limit the returned cells using a filter (as in the current state of this PR)

I'm still not sure whether 1 or 2 is the better way wrt "all the things".

Reply:
It seems, there is also the "delete-then-write" option: https://cloud.google.com/bigtable/docs/keep-only-latest-value#delete_then_write

Have you considered that?